tests/units/test_lib.rb

```diff
@@ -83,6 +83,7 @@ def test_commit_with_no_verify
 
   def test_checkout
     assert(@lib.checkout('test_checkout_b',{:new_branch=>true}))
+    assert(@lib.checkout('test_checkout_b2', {new_branch: true, start_point: 'master'}))
```

Comment:
I know that this is duplicating the tests that are already here in `test_checkout`, but it doesn't really test what you are changing. Only that the command (whatever command was constructed -- even if it is wrong) succeeded.

Reply:
In addition to this test, can you add a separate test (something like `test_checkout_with_start_point`) that tests that the command is constructed correctly like this one:

https://github.com/bradleybuda/ruby-git/blob/master/tests/units/test_commit_with_gpg.rb#L14